cmd/create_deploy.go

```diff
@@ -4,15 +4,16 @@ import (
 	"fmt"
 	"github.com/nullstone-io/deployment-sdk/app"
 	"gopkg.in/nullstone-io/go-api-client.v0"
+	"gopkg.in/nullstone-io/go-api-client.v0/types"
 )
 
-func CreateDeploy(nsConfig api.Config, appDetails app.Details, version string) error {
+func CreateDeploy(nsConfig api.Config, appDetails app.Details, version string) (*types.Deploy, error) {
 	client := api.Client{Config: nsConfig}
-	result, err := client.Deploys().Create(appDetails.App.StackId, appDetails.App.Id, appDetails.Env.Id, version)
+	newDeploy, err := client.Deploys().Create(appDetails.App.StackId, appDetails.App.Id, appDetails.Env.Id, version)
 	if err != nil {
-		return fmt.Errorf("error updating app version: %w", err)
-	} else if result == nil {
-		return fmt.Errorf("could not find application environment")
+		return nil, fmt.Errorf("error updating app version: %w", err)
+	} else if newDeploy == nil {
+		return nil, fmt.Errorf("could not find application environment")
```

Comment:
I don't think this error message is correct, is it? Shouldn't it be something like "unable to create a deploy?"

Reply:
Good catch!